pkg/worker/worker.go

```diff
@@ -875,10 +882,9 @@ func (s *Worker) shutdown() error {
 	}
 
 	s.cancel()
-
-	err := s.storage.Unmount(s.config.Storage.FilesystemPath)
+	err := s.storageManager.RemoveWorkerStorage()
```

Comment:
What is the benefit of this if we already cleaned up the mounts by container ID?